stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
-                        aria-haspopup="true" aria-expanded="false" aria-label="More" onclick="onControlClick('ggiuhwer');"></button>
-                    </div>
-                    <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="ggiuhwer">
-                        <nav class="fd-menu">
-                            <ul class="fd-menu__list fd-menu__list--no-shadow">
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Edit</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Lock</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Duplicate</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Delete</span>
-                                    </a>
-                                </li>
-                            </ul>
-                        </nav>
-                    </div>
-                </div>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container">
+            <div class="fd-tree__content">
+                <span class="fd-tree__text">Level 1</span>
             </div>
         </div>
     </li>
 </ul>
 `;
 
-export const expandedTree = () => `
-<div class="fd-tree fd-tree--header">
-    <div class="fd-tree__row fd-tree__row--header">
-        <div class="fd-tree__col fd-tree__col--control">
-            <button class="fd-tree__control" aria-label="Expand all" aria-controls="tWsod582" aria-pressed="false"></button>
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Status
-        </div>
-        <div class="fd-tree__col fd-tree__col--actions">
+treeWithExpandedLevel3AndHighlightIndicator.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: `
+Indentations are calculated with 0.5rem up to level 6.
+When Level 3 is expanded to show Level 4:
+
+- Level 2 changes to be indented by 0.5rem
+- Level 3 changes to be indented by 1rem
+- Level 4 changes to be indented by 1.5rem
```

Comment:
Again, you can say "indents" as the verb here. So, "Level 2 indents by 0.5rem" :)